src/Symfony/Component/Translation/Translatable.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation;
+
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+// Load the global t() function
+require_once __DIR__.'/Resources/functions/translatable.php';
```

Comment:
isn't it loaded by composer ?

Reply:
see #37989

Reply:
@fabpot does that mean https://github.com/symfony/symfony/blob/master/src/Symfony/Component/VarDumper/VarDumper.php#L28 is also unnecessary?